aries-site/src/pages/templates/page-layouts.js

```diff
@@ -16,10 +50,128 @@ const PageLayouts = () => {
         description={pageDetails.seoDescription}
         canonicalUrl="https://design-system.hpe.design/templates/page-layouts"
       />
-      <ContentSection>
-        <Subsection name={title} level={1} topic={topic} />
-      </ContentSection>
-      <ComingSoon />
+      <ResponsiveContext.Consumer>
+        {size => (
+          <>
+            <ContentSection>
+              <Subsection name={title} level={1} topic={topic}>
+                <SubsectionText>{pageDetails.description}</SubsectionText>
+                <SubsectionText>
+                  Here are some common layouts to get you started:
+                </SubsectionText>
+                <Grid
```

Comment:
Since this primary component is getting quite lengthy, and this portion is a bit more involved, the readability of the intent/flow of the page is starting to have too much going on. Consider breaking out the Grid portion as a "LayoutPreview" or "LayoutIndex" sub component, that way the preview implementation can be compartmentalizes and digested more easily.

Reply:
Done.